internal/runners/activate/activate.go

```diff
@@ -72,8 +72,17 @@ func (r *Activate) run(params *ActivateParams) error {
 	alreadyActivated := subshell.IsActivated()
 	if alreadyActivated {
 		if !params.Default {
-			return locale.NewInputError("err_already_activated", "You cannot activate a new project when you are already in an activated state.")
+			err := locale.NewInputError("err_already_activated",
+				"You cannot activate a new project when you "+
+					"are already in an activated state.",
+			)
+			tipMsg := locale.Tl("err_tip_exit_activated",
+				"To exit your activated state simply close your "+
+					"current shell by running [ACTIONABLE]exit[/RESET].",
+			)
+			return errs.AddTips(err, tipMsg)
```

Comment:
The tip and the error are not the same:

https://www.pivotaltracker.com/n/projects/2203557/stories/175600387

Reply:
Sorry, I read that wrong. Fixed!